src/yellow/components/button/_index.scss

```diff
@@ -2,126 +2,158 @@
 
 .btn {
   border: 0;
+  border-radius: $size-border-radius-pill;
   cursor: pointer;
-  display: inline-block;
   font-weight: $font-style-weight-bold;
-  letter-spacing: .3px;
-  line-height: 1;
+  min-width: 280px;
   text-align: center;
   text-transform: uppercase;
-  transition: border-color .2s ease-in, box-shadow .2s ease-in, background-color .2s ease-in, color .3s ease;
+  transition: all .2s linear;
 
-  &__primary {
-    border-radius: $size-border-radius-pill;
-    transition: all .2s linear;
+  &--small {
+    font-size: $size-font-xxs;
+    padding: $size-spacing-xxs $size-spacing-m;
+  }
+
+  &--large {
+    font-size: $size-font-xs;
+    padding: $size-spacing-xs $size-spacing-l;
+  }
+
+  &--primary {
```

Comment:
What is the difference of the primary to the default version? The primary shouldn't be the default version?

Reply:
To name the buttons I followed a lot what is in Figma designed by the designers. I thought it would look better this way when assembling the layouts.

Reply:
Now I saw the layout! It make sense, following the ubiquitous language principle! 👏🏽

Reply:
Looking at storybook, the hover state of inverse primary button, the font color seems wrong, compared to Figma. I think the font color should be black in this state. 🔍
<img width="330" alt="Captura de Tela 2019-12-11 às 15 22 09" src="https://user-images.githubusercontent.com/4427237/70648832-b212c480-1c2a-11ea-9db1-de00be62dd97.png">

Reply:
You're right, I'll adjust. Thank you!